redwood-protocol-widget/src/commonMain/kotlin/app/cash/redwood/protocol/widget/ProtocolBridge.kt

```diff
@@ -90,8 +93,11 @@ public class ProtocolBridge<W : Any>(
           }
         }
         is ModifierChange -> {
+          val modifier = change.elements.fold<_, Modifier>(Modifier) { modifier, element ->
+            modifier.then(factory.createModifier(element))
+          }
```

Comment:
My first use of [underscore type operator](https://kotlinlang.org/docs/whatsnew17.html#underscore-operator-for-type-arguments) and I love it.

Reply:
NEAT